clang/test/SemaSYCL/num_simd_work_items_device.cpp

```diff
@@ -131,9 +131,7 @@ struct TRIFuncObjBad8 {
 [[intel::num_simd_work_items(2)]] // expected-error{{'num_simd_work_items' attribute must evenly divide the work-group size for the 'reqd_work_group_size' attribute}}
 [[sycl::reqd_work_group_size(4, 2, 3)]] void func1(); // expected-note{{conflicting attribute is here}}
 
-[[intel::reqd_work_group_size(4, 2, 3)]] // expected-note{{conflicting attribute is here}} \
-                                         // expected-warning {{attribute 'intel::reqd_work_group_size' is deprecated}} \
-                                         // expected-note {{did you mean to use 'sycl::reqd_work_group_size' instead?}}
+[[sycl::reqd_work_group_size(4, 2, 3)]] // expected-note{{conflicting attribute is here}}
```

Comment:
What does this conflict with still?

Reply:
Thanks @premanandrao for the reviews.

>>I think it would make sense to have at least one test that uses the removed spelling and now shows an error.

I have added a test that shows warning for removed attribute spelling
[[intel::reqd_work_group_size(4, 2, 9)]]        // expected-warning{{unknown attribute 'reqd_work_group_size' ignored}}

>>What does this conflict with still?

This conflicts with [[intel::num_simd_work_items()]] attribute.
// If the declaration has a [[sycl::reqd_work_group_size]] attribute, tests that check
// if the work group size attribute argument (the last argument)
// can be evenly divided by the [[intel::num_simd_work_items()]] attribute.

[[sycl::reqd_work_group_size(4, 2, 3)]]         // expected-note{{conflicting attribute is here}}
 [[intel::num_simd_work_items(2)]] void func2(); // expected-error{{'num_simd_work_items' attribute must evenly divide the work-group size for the 'reqd_work_group_size' attribute}}

Reply:
> This conflicts with [[intel::num_simd_work_items()]] attribute.

Thanks for explaining.